api/openapi.yaml

```diff
@@ -1876,6 +2066,9 @@ components:
           type: integer
         summary:
           $ref: '#/components/schemas/ScanFindingsSummary'
+        providerID:
+          type: string
+          description: Provider ID of the provider responsible for picking up and running the asset scan.
```

Comment:
I think this should really be a relationship to a provider (`$ref: "...../ProviderRelationship`) not simply the id to allow it to be $expand-able in the API.